src/components/FilterListItem/FilterListItem.jsx

```diff
@@ -21,15 +21,15 @@ class FilterListItem extends React.Component {
               className="button is-link"
               title="Edit filter"
             >
-              <span className="octicon octicon-pencil"></span>
+              ✏️
             </button>
             <button
               onClick={() => this.props.delete(this.props.filter)}
               type="button"
               className="button is-link"
               title="Delete filter"
             >
-              <span className="octicon octicon-trashcan"></span>
+              ❌
```

Comment:
Re: wanting ignore vs. delete to be different icons - I think in the context of this app, they are basically the same functionality - "I don't want to see this in my list". The notifications are from GitHub, so we don't want to delete them outright, but the only real difference is that a filter can't be un-deleted. 

so... unless there's a stronger icon than X, we should reserve X for things that are permanent. Archiving is a less permanent decision, so its icon should be less severe. Ideas for that icon?